backend/src/routes/groupRoutes.ts

```diff
@@ -0,0 +1,83 @@
+import { Request, Response, Router } from 'express';
+import {
+  addGamerToGroup,
+  createGroup,
+  deleteGroup,
+  getGroups,
+  removeGamerFromGroup,
+} from '../controllers/groupController';
+import { PayloadJWT, getPayload } from '../services/authenticationService';
+
+const router = Router();
+
+router.get('/', async (req: Request, res: Response) => {
+  try {
+    const authorization = req.headers.authorization;
+    if (!authorization) {
+      return res.status(500).json('No authorization header');
+    }
+    const accessToken = authorization.replace('AccessToken ', '');
+    if (!accessToken) {
+      return res.status(500).send('No access token');
+    }
+
+    const payload: PayloadJWT = getPayload(accessToken);
+    const groups = await getGroups({ gamerId: payload.id });
+    res.status(200).json(groups);
+  } catch (e) {
+    res.status(500).json({ error: `There isn't any existing group` });
+  }
+});
+
+router.post('/new', async (req: Request, res: Response) => {
+  try {
+    const authorization = req.headers.authorization;
+    if (!authorization) {
+      return res.status(500).json('No authorization header');
+    }
+    const accessToken = authorization.replace('AccessToken ', '');
```

Comment:
If you use the authentication middleware, you can get the accessToken just by typing req.accessToken, and no need of check on him :)